packages/docs/docs/api/fhir/operations/patient-update-account.md

```diff
@@ -0,0 +1,57 @@
+---
+sidebar_position: 5.1
+---
+
+# Patient $update-account Operation
+
+Medplum implements a custom `$update-account` operation for Patient resources to manage account associations.
+
+This operation updates the account references for all resources in the patient compartment to match the patient's accounts. This is useful when you need to ensure consistent account access across all resources related to a patient.
```

Comment:
We may wish to note that "all" isn't totally accurate here, and only the first 1,000 resources in the compartment will be updated